<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QVTKWidget" name="view1" native="true"/>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="slider1">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBothSides</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox1">
             <property name="maximum">
              <double>1500.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QVTKWidget" name="view2" native="true"/>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="slider2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox2">
             <property name="maximum">
              <double>512.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QVTKWidget" name="view3" native="true"/>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="slider3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox3">
             <property name="maximum">
              <double>512.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QVTKWidget" name="view4" native="true"/>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="slider4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGetCL"/>
    <addaction name="actionSetThinkness"/>
    <addaction name="actionSurface"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>camera</string>
    </property>
    <addaction name="actionRoll"/>
    <addaction name="actionAzimuth"/>
    <addaction name="actionElevation"/>
   </widget>
   <widget class="QMenu" name="menuDistance">
    <property name="title">
     <string>distance</string>
    </property>
    <addaction name="actionDiswidget"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuDistance"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGetCL">
   <property name="text">
    <string>getCL</string>
   </property>
  </action>
  <action name="actionRoll">
   <property name="text">
    <string>roll</string>
   </property>
  </action>
  <action name="actionAzimuth">
   <property name="text">
    <string>azimuth</string>
   </property>
  </action>
  <action name="actionElevation">
   <property name="text">
    <string>elevation</string>
   </property>
  </action>
  <action name="actionYaw">
   <property name="text">
    <string>yaw</string>
   </property>
  </action>
  <action name="actionPitch">
   <property name="text">
    <string>pitch</string>
   </property>
  </action>
  <action name="actionRoll2">
   <property name="text">
    <string>roll2</string>
   </property>
  </action>
  <action name="actionSetThinkness">
   <property name="text">
    <string>setThinkness</string>
   </property>
  </action>
  <action name="actionSurface">
   <property name="text">
    <string>surface</string>
   </property>
  </action>
  <action name="actionDiswidget">
   <property name="text">
    <string>diswidget</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
